autoscheduler/scraper/management/commands/scrape_courses.py

```diff
@@ -232,9 +234,12 @@ def save_models(instructors: List[Instructor], sections: List[Section], # pylint
 
     start = time.time()
     with transaction.atomic():
+        grades_to_resave = list(Grades.objects.filter(section__in=sections))
```

Comment:
idk why it took me so long to figure out what this was doing considering it's literally called `grades_to_resave`, but can you move/copy the comment about why we do this here instead of where they're resaved

Reply:
Oh yeah also you should note that it calls `list()` so it's forced to evaluate the queryset here instead of later (when it'll be empty), since this is a little unintuitive